pkg/virtctl/pause/pause.go

```diff
@@ -61,6 +65,7 @@ Second argument is the name of the resource.`,
 			return c.Run(args)
 		},
 	}
+	cmd.Flags().BoolVar(&dryRun, "dry-run", false, "--dry-run=false: Flag used to set whether perform a dry run or not. If true the command will executed without performing any changes.")
```

Comment:
Missing `to` after `whether`, like in the other PR :)
Also, "will **be** executed" maybe?

Reply:
I'm really tenacious doing the same mistake :) Thanks, will fix

Reply:
Thanks again for the review. Fixed